.github/workflows/test.yaml

```diff
@@ -20,3 +20,7 @@ jobs:
       uses: actions/checkout@v4
     - name: Test
       run: sudo go test -v ./...
+    - name: Upload coverage reports to Codecov
+      uses: codecov/codecov-action@v3
+      env:
+        CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
I think it's working now.

https://app.codecov.io/gh/msteinert/pam/pull/19

Reply:
Not properly, since you're including a coverage out file in the repo, drop it and change the test.yaml to save it in a place and make the codecov job to upload it.